src/WebServer_AdvancedConfigPage.ino

```diff
@@ -126,9 +126,9 @@ void handle_advanced() {
   addFormSubHeader(F("Location Settings"));
   addFormNumericBox(F("Timezone Offset (UTC +)"), F("timezone"), Settings.TimeZone, -720, 840); // UTC-12H ... UTC+14h
   addUnit(F("minutes"));
-  addFormFloatNumberBox(F("Latitude"), F("latitude"), Settings.Latitude, -90.0, 90.0);
+  addFormFloatNumberBox(F("Latitude"), F("latitude"), Settings.Latitude, -90.0, 90.0f);
```

Comment:
Shouldn't the negative values also get the `f` marker?

Reply:
Yep, missed those.
It is very likely I did miss a few more as I started with some query replace and then worked further on those files that had changes.